token-holders/how-to-stake.md

```diff
@@ -0,0 +1,40 @@
+{% hint style="info" %}
+Staked tokens can't be withdrawn immediately, make sure you understand the [rules](staking-rules.md) before staking your tokens.
+{% endhint %}
+
+## The staking website
```

Comment:
this title doesn't seem necessary -- also there's no title in the governance counterpart so we should probably standardize it

Reply:
agree with this